WcaOnRails/spec/factories/rounds.rb

```diff
@@ -8,6 +8,7 @@
       format_id "a"
     end
 
+    roundTypeId "1"
```

Comment:
Interestingly I forgot that tests is the only place were we don't delete+recreate rounds to update them.

Maybe then a better approach would be to create the roundTypeId upon creation?
It would also needs to be updated if updating the cutoff, or if the `competition_events` gets more/less rounds...

I'd like some other opinions as our only use case in real life always destroy then recreate the round to update it.